great_expectations/expectations/core/expect_column_values_to_be_of_type.py

```diff
@@ -413,18 +413,29 @@ def _validate_sqlalchemy(self, actual_column_type, expected_type, execution_engi
 
         if expected_type is None:
             success = True
+            # Default result for None expected_type;
+            # observed_value might be the type name or actual_column_type itself
+            return {
```

Comment:
Is this branch tested? codecov seems to think not which means it wasn't tested before either. 
While not necessary it might be nice to add a test, the logic here is a bit different than before.